builtin/settingtypes.txt

```diff
@@ -1324,6 +1324,15 @@ secure.trusted_mods (Trusted mods) string
 #    allow them to upload and download data to/from the internet.
 secure.http_mods (HTTP mods) string
 
+#    Prevent servers from running custom shaders on your client
+#    type: bool
+secure.enable_shader_security (Enable shader security) bool true
```

Comment:
1. This setting is badly named - it enables server-sent shaders, not shader security.
2. Does it need to exist, when `trusted_shader_servers` exists?

Reply:
secure.enable_shader_security == !secure.enable_server_sent_shaders

Reply:
Yes, I mean to invert the meaning too